### ISSUE TITLE
Add method for updating JWT

### ISSUE THREAD

**Issue by Mangatt** [feathersjs/feathers-authentication#260]
Hi,
it would be great if server could issue new token with new data - new expiration or session-like data. Now i couldn't do either, token have to expire by itself and then can be issued and session-like data can be saved in connection object, but are lost after connection and reconnection.

**Comment by ekryski** [comment]
@Mangatt You can always make a request using a valid token and it will return a new one, albeit with the same data and expiration, unless you setup a `before` hook on the `auth/token` service to alter the JWT payload.

We have some fixes coming in the 0.8 branch that will make it easier to issue tokens with different options and different payloads.

**Comment by ekryski** [comment]
With the new 1.0.0 version of auth you can create a new JWT at any time by calling:

```js
const options = app.get('auth');
app.passport.createJWT(payload, options)
```

or providing whatever custom JWT options you would like:

```js
const options = {
  secret: 'your secret',
  jwt: {
   // your custom options. See https://github.com/auth0/node-jsonwebtoken
  }
};
app.passport.createJWT(payload, options)
```

So it is entirely up to you when you would like to do it.
